content/swift/concepts/protocols/terms/encodable/encodable.md

```diff
@@ -26,6 +34,63 @@ struct myStruct: Encodable {
 }
 ```
 
+## Conforming Types:
```

Comment:
This entire section is a large infodump in the middle of the entry. Docs entries are supposed to be relatively short and easy to digest. I don't think this list is adding anything to the entry. I'd remove it.

Reply:
I deleted all lines. I think, that I had too little knowledge about Codecademy docs and its intentions when I started to work on this article. So I simply added Information, that seemed to be important to me. It was a demanded exercise in the iOS development career path and I could not find out in the description what exactly should be covered in the articles. So I simply jumped in.

Reply:
I made all changes in my local repo. In case that nearly all of my adds do not really match, I would suggest to simply delete my branch. If you do not agree with me please let me know what to do next. Thank you for your understanding.